sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Filtering/Filter.cs

```diff
@@ -447,6 +447,21 @@ private Expression ProduceComparatorExpressionForSingleFieldCondition(Expression
 
                             Type enumUnderlyingType = fieldType.GetTypeInfo().GetEnumUnderlyingType();
 
+                            static Type GetNullableType(Type inputType) => inputType switch
+                            {
+                                _ when inputType == typeof(sbyte) => typeof(sbyte?),
+                                _ when inputType == typeof(short) => typeof(short?),
+                                _ when inputType == typeof(int) => typeof(int?),
+                                _ when inputType == typeof(long) => typeof(long?),
+                                _ when inputType == typeof(byte) => typeof(byte?),
+                                _ when inputType == typeof(ushort) => typeof(ushort?),
+                                _ when inputType == typeof(uint) => typeof(uint?),
+                                _ when inputType == typeof(ulong) => typeof(ulong?),
+                                _ when inputType == typeof(float) => typeof(float?),
+                                _ when inputType == typeof(double) => typeof(double?),
+                                _ => throw new ArgumentException($"Cannot create a nullable type for {inputType.FullName}."),
+                            };
```

Comment:
This block matches the case statements at lines 423-432.

Reply:
Sounds like this would be a good comment to add in code. For future readers to see as well.